.github/workflows/main.yml

```diff
@@ -0,0 +1,119 @@
+name: Flowbroker
+
+# Triggered after a branch is created, a tag or a branch is pushed or a PR
+on:
+  create:
+  push:
+  pull_request:
+
+jobs:
+  npm-build:
+    name: npm Build
+    # If the commit message contains 'ci skip', it is not necessary to run this job
+    if: contains(github.event.head_commit.message, 'ci skip') == false
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [8.x]
+    steps:
+
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm ci, build and check licenses
+      run: |
+        cd lib
+        npm ci
+        npm run build --if-present
+        npm run check-licenses
+
+  docker-compose:
+    needs: npm-build
+    name: Docker Compose build
+    runs-on: ubuntu-18.04
+    steps:
+
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: Build the stack and test
+      run: |
+        export FLOWBROKER_NETWORK=flowbroker
+        docker build -t flowbroker -f orchestrator.docker .
+        docker build -t flowbroker-context-manager -f contextManager.docker .
+        docker build -t tests-flowbroker -f ./tests/Dockerfile .
+        docker-compose -f tests/docker-compose.yml up -d kafka rabbitmq data-broker mongodb auth postgres-users device-manager device-manager-redis
+        sleep 10
+        docker-compose -f tests/docker-compose.yml up -d flowbroker flowbroker-context-manager flowbroker-redis
+        sleep 10
+        docker-compose -f ./tests/docker-compose.yml run --rm tester * --level=debug
+
+  docker-hub:
+    needs: docker-compose
+    name: Build and Publish image to DockerHub
+    runs-on: ubuntu-18.04
+    # If this is a PR, it won't run
+    if: contains(github.ref, 'refs/pull/') == false
+    steps:
+
+    - name: Owner Repository
+      id: get_owner
+      run: |
+        export OWNER=$(echo ${{ github.repository }} | cut -f1 -d"/")
+        echo Current Owner is $OWNER
+        echo ::set-output name=owner::$OWNER
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Build and Push
+      uses: elgohr/Publish-Docker-Github-Action@2.16
+      with:
+        name: ${{ steps.get_owner.outputs.owner }}/flowbroker
```

Comment:
It's necessary to build the image `dojot/flowbroker-context-manager` too
Always look at /travis/publish.sh to see what it did.